examples/AffinePlaneCurve.jl

```diff
@@ -611,6 +611,18 @@ function issmooth_curve(C::AffinePlaneCurve)
   end
 end
 
+################################################################################
+# hash functions
+
+function Oscar.hash(C::AffinePlaneCurve, h::UInt)
+  F = 1//lc(C.eq)*C.eq
+  return hash(F, h)
+end
+
+function Oscar.hash(C::AffinePlaneCurve)
```

Comment:
This one is not needed, it's automatically defined for all types.

Reply:
For the curious, this is well-hidden here: https://docs.julialang.org/en/v1/base/base/#Base.hash.